### PR TITLE
Remove link to the constants DONE and UNSENT

### PR DESCRIPTION
**Pull request by teoli2003**
We don't write separate pages for constants. Transforms the links to a simple text.

### CODE REVIEW

File: files/en-us/web/api/xmlhttprequest/abort/index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ browser-compat: api.XMLHttpRequest.abort
 The **`XMLHttpRequest.abort()`** method aborts the request if
 it has already been sent. When a request is aborted, its
 {{domxref("XMLHttpRequest.readyState", "readyState")}} is changed to
-{{domxref("XMLHttpRequest.UNSENT")}} (0) and the request's
+`UNSENT` (0) and the request's
```
Comment (Josh-Cena) [inline files/en-us/web/api/xmlhttprequest/abort/index.md:13]: `XMLHttpRequest.UNSENT` is an actual JS entity. Even if we don't have a page for it (see also: https://github.com/mdn/content/issues/9156, I think we should at least have a mention of it on the `XMLHttpRequest` landing page), we should keep the `XMLHttpRequest.UNSENT` qualified name.

**Comment by teoli2003** [comment]
Yes, you are right, let's keep the prefix.

**Review by Josh-Cena — APPROVED** [review]
If it's an established decision to not document constants/enums, then this LGTM.

**Comment by teoli2003** [comment]
It is; we don't document them on their page. (The state machine should, directly or indirectly, be explained somehow, somewhere).

**Pull request merged**